pages/index.js

```diff
@@ -3,17 +3,59 @@ import Footer from '../components/Footer';
 import Jumbotron from '../components/Jumbotron';
 import GameCard from '../components/GameCard';
 import SeeMore from '../components/SeeMore';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
 
 export default function Home() {
+
+  const [games, setGame] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(process.env.NEXT_PUBLIC_BASE_URL + "games/listing")
```

Comment:
Maaf, aku kurang paham juga sih mas, tapi apakah ini pakenya "games/listing" atau "games/landing"?

Reply:
Sip. Thank you, Mas.